### DISCUSSION TITLE
Obtaining already compiled fonts

### DISCUSSION THREAD

**Discussion by theHedster** [Shavian-info/interalia#4] [Q&A]
I'm learning to use Shavian phonemic alphabet that allows you to spell words phonetically so what you see is what you say. 

There are a number of fonts for Shavian that contain the glyph set but usually they are for only one or two styles: upright, or upright and slanted, or slanted. None have the basic four styles: regular, italic, bold regular, and bold italic.

There is a fork* of Inter called Inter Alia which contains the Shavian glyph set in multiple masters. I just want a single *.otf file that contains the basic four styles in a weight suitable for setting book text. If the display styles could also be available in a separate *.otf file that would be good.

For a complete novice, the instructions for compling a font set are mind boggling. Can someone please point me to *.otf files from Inter Alia that have already been compiled with the basic four styles and posted for download?

Or instructions for running the tool chain to get the desired result. I would then offer the resulting files to shavian.info to make them available to r Shavian users.

**Comment by theHedster** [comment]
* Not sure if Inter Alia is a fork or just the name of the project that developed the Shavian glyphs. PS: I used to work in printing and publishing when it was done by real men with hot-metal typesetting, etc  So I know a lot of typography jargon but not so good on developer jargon!

**Reply by Shavian-info** [reply]
Sorry for the delayed response. I only just saw this. The font files may be found in the folder docs/font-files. I don't believe it is possible to include regular, bold, italic and bold-italic in one OTF. If you have an app that allows you to use the variable font, you could install that. Otherwise, you'll need to install the OTF for the relevant weights/styles you'd like to use.
